### ISSUE TITLE
Hasura on Cloudfoundry Platform

### ISSUE THREAD

**Issue by mthota15** [hasura/graphql-engine#2891]
Is possible to run Hasura on  Pivotal Cloudfoundry Platform(PCF)? If so, any guidance on how to deploy into PCF?

**Comment by dsandip** [comment]
@mthota15 Yes, it is possible to run Hasura on Cloudfoundry and several users in the [community](https://discord.gg/hasura) have done so. I believe CF docs address the topic of [running Docker apps](https://docs.pivotal.io/pivotalcf/2-6/devguide/deploy-apps/push-docker.html). Changing the scope of this issue to document this process in a Hasura guide.

**Comment by pdominique** [comment]
This might help -> https://blogs.sap.com/2019/11/06/instant-realtime-graphql-engine-on-cloud-foundry-with-postgresql-and-docker/

**Comment by marionschleifer** [comment]
This can be a deployment guide [here](https://hasura.io/docs/1.0/graphql/core/deployment/deployment-guides/index.html).

**Comment by Smartmind12** [comment]
Hey, I would like to work on this as a part of Hacktoberfest 2022, Could you assign it to me?

**Comment by Stefmore02** [comment]
@Smartmind12 thank you for your interest! FYI - We don't assign issues so feel free to pick this up. The PR that is submitted first will also be the one to be reviewed first 🙂 so far, nobody has raised a PR on this.

**Comment by Smartmind12** [comment]
@Stefmore02 @marionschleifer Kindly review my submission and propose recommendations if any!
Also, requesting you to assign the "Hacktoberfest" label if possible. Thanks!

**Comment by Stefmore02** [comment]
@Smartmind12 thank you for your submission, your PR will be reviewed within 2 weeks 🤓
